README.md

```diff
@@ -45,14 +45,19 @@ directory `build/chromium`.
 
 ## Getting the Code
 
+Next, install Node.js via the [official package](http://nodejs.org) or via
+[nvm](https://github.com/creationix/nvm).
+
 To get a local copy of the current code, clone it using git:
 
     $ git clone git://github.com/mozilla/pdf.js.git
     $ cd pdf.js
 
-Next, install Node.js via the [official package](http://nodejs.org) or via
-[nvm](https://github.com/creationix/nvm). You need to install the gulp package
-globally (see also [gulp's getting started](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md#getting-started)):
+The package is also distributed via npm:
```

Comment:
Create separate section for this e.g. "Using the Core Library in a Web Application", and revert any change in "Getting the Code" section.